services/integration/src/controllers/constitution/document.js

```diff
@@ -0,0 +1,32 @@
+const Document = require('../../models/constitution/document.model');
+const Controller = require('../controller');
+const { asyncHandler } = require('../../utils');
+
+const create = asyncHandler(async (req, res) => {
+  const document = req.body;
```

Comment:
This variable should be called documents... because you are receiving an array of documents, not a single document